src/Apps/ViewingRoom/Routes/Statement/ViewingRoomStatementRoute.tsx

```diff
@@ -34,6 +35,8 @@ const StatementRoute: React.FC<ViewingRoomStatementRouteProps> = ({
           >
             View works
           </Button>
+          <ViewingRoomPullQuote viewingRoom={viewingRoom} />
```

Comment:
@damassi - I think it's logical to add spacing on this level and not inside the `quote` component itself. This way we can shuffle things around at that level not worrying of needing to change each component.

By that logic I also think that the `Button` right above should be part of `ViewingRoomWorks` since it should always follow the works row display and if we move those - the button should follow. No?

Reply:
That makes sense!